tests/Behat/Resources/services.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sylius.behat.context.setup.reorder" class="Tests\Sylius\CustomerReorderPlugin\Behat\Context\Reorder\ReorderContext" public="false">
```

Comment:
I would say it should be sth like `sylius_customer_reoder.behat.context.setup.reorder`